### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD031.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#34] [General]
# https://blog.3place.cn/3place/fsd/FSD031.html

https://blog.3place.cn/3place/fsd/FSD031.html

**Comment by picklecai** [comment]
灵魂和肉体这样巧妙地调和，
紧紧结合，好像永不要离分，
却为什么每天总不能安分。

这是说灵魂每时每刻不得满足吗？

**Comment by picklecai** [comment]
荷蒙库路斯
（惊异。）
有意思〔235〕！——
（烧瓶从瓦格纳手里滑脱，飞到浮士德头上飘荡，照耀着他。）
四周真美！——清清的流水
在茂林之中；妇女们把衣服脱下，

荷蒙库路斯能看见浮士德的梦境。

**Comment by picklecai** [comment]
他们自称是争取自由人权，
细看无非是奴隶对奴隶之战。

若没有这样的错觉，人生百年又该如何难过！

**Comment by picklecai** [comment]
会发现i字上面的一点

这句话是什么意思？
